pkg/util/config/feature_gate.go

```diff
@@ -37,6 +37,8 @@ const (
 	// values override the default set by allAlpha.
 	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
 	allAlpha = "allAlpha"
+
+	externalTrafficLocalOnly = "externalTrafficLocalOnly"
```

Comment:
Jeff - the problem is that I can't easily tell if the feature-specific flag
"overrides" - it return a bool.  Was that bool the default value (don't
override) or explicit (override) ?

I have a PR in flight to do this:

if allAlphas is set, explicitly set all of the alpha flags to that value.
This allows `allAlphas=true,myFeature=false` - you ok with that?

On Mon, Aug 22, 2016 at 10:42 AM, Jeff Lowdermilk <notifications@github.com>
wrote:

> In pkg/util/config/feature_gate.go
> <https://github.com/kubernetes/kubernetes/pull/29409#discussion_r75723012>
> :
>
> > @@ -37,6 +37,8 @@ const (
> >  	// values override the default set by allAlpha.
> >  	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
> >  	allAlpha = "allAlpha"
> > +
> > +	externalTrafficLocalOnly = "externalTrafficLocalOnly"
>
> Should check AllAlpha gate as default. If the feature-specific flag is
> provided, it overrides.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29409/files/59883dc63dbea4bc31ef2ef85d3274a0ea383e78..9875b3b6f8892cf2c1e4c0aa2009b5161448c094#r75723012>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVBBeRheM1a7LWWhd7lAGnVI0sJ4lks5qid96gaJpZM4JSUXH>
> .
>